### PR TITLE
Lowers rot disease chance

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Lowers rot disease chance. That's it.

## Why It's Good For The Game

Every round now turns into a shitshow with disease outbreak every 10 minutes. Not cool.

## Changelog
:cl:
balance: Lowered the chance of disease creation by ~60%
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -266,7 +266,7 @@
 	if(breath_gases[/datum/gas/miasma])
 		var/miasma_partialpressure = (breath_gases[/datum/gas/miasma][MOLES]/breath.total_moles())*breath_pressure
 
-		if(prob(1 * miasma_partialpressure))
+		if(prob(0.2 * miasma_partialpressure))
```
Comment (LemonInTheDark) [inline code/modules/mob/living/carbon/life.dm:269]: why are you touching miasma?

File: code/datums/components/rot.dm [hunk 2]
```diff
@@ -119,7 +119,7 @@
 	if(world.time - start_time < start_delay)
 		return
 
-	if(!prob(strength * 8))
+	if(!prob(strength * 2))
```
Comment (LemonInTheDark) [inline code/datums/components/rot.dm:122]: you should make this like 3, if 8 is too high 2 is gonna be a bit low

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (SmArtKar) [reply code/modules/mob/living/carbon/life.dm:269]: From what I know, miasma is causing most of the issues.

Comment (LemonInTheDark) [reply code/modules/mob/living/carbon/life.dm:269]: you should read the pr that made these changes closer, because it's not

**Comment by ATH1909** [comment]
tbh, this seems like more of a fundamental issue with the way that rot is handled now than something that can be fixed by just tweaking some odds.

potato shut down your other PR, though, so :shrug:.

**Comment by moocowswag** [comment]
It definitely is a fundamental flaw and it will take alot of changes to get things right.

Take ash walkers for example, this wont really help them much at all but they are sort of designed to not be able to get spontanous diseases in the first place without crew intervention thanks to them not needing to breath miasma but that no breathing trait no longer protects them.

**Comment by Yawet330** [comment]
This does not respect armor still, meaning biosuits ecetera still do not work. There is currently no way to stop this.

**Comment by LemonInTheDark** [comment]
> 
> 
> tbh, this seems like more of a fundamental issue with the way that rot is handled now than something that can be fixed by just tweaking some odds.
> 
> potato shut down your other PR, though, so 🤷.

Are you talking about my fuckup when it comes to target_zones? or something else

**Comment by LemonInTheDark** [comment]
> 
> 
> This does not respect armor still, meaning biosuits ecetera still do not work. There is currently no way to stop this.

what?

**Comment by ATH1909** [comment]
> This does not respect armor still, meaning biosuits ecetera still do not work. There is currently no way to stop this.

you 100% sure about this? doesn't rot call ContactContractDisease()?

**Comment by carshalash** [comment]
Can this please get speed merged?  This shit isn't fun.

**Pull request merged**